README.md

```diff
@@ -105,7 +105,8 @@ run the grpc_tools generator:
 ```shell
 cd $HOME
 git clone https://github.com/googleapis/googleapis
-cd storage-testbench
+patch -p1 <$HOME/storage-testbench/patches/rpo-and-cdr.patch
```

Comment:
Was this expected to be merged?

Reply:
Yes, that explains how to refresh the proto and grpc-generated files. Yes, we are patching the protos locally. Eventually the upstream protos will include the necessary fields and we can remove the patch.